src/v2/components/WorkspaceMembers/index.tsx

```diff
@@ -0,0 +1,244 @@
+										onClick={
+											() => {
+												setPage(0)
+												setSelectedUserTypeIdx(index)
+											}
+										}
+										bg={index === selectedUserTypeIdx ? '#1F1F32' : '#E0E0EC'}
+										fontSize={18}
+										borderRadius={2}
+										paddingTop={1}
+										paddingBottom={1}
+										letterSpacing={-1}
+										textTransform={'none'}
+										cursor={'pointer'}
+										color={index === selectedUserTypeIdx ? '#E0E0EC' : '#1F1F32'}>
+										{userType.name}
+									</Tag>
+								</WrapItem>
+							),
+							)
+						}
+
+					</Wrap>
+					<Spacer />
+					<Button
+						onClick={() => setIsInviteModalOpen(true)}
+						leftIcon={<LinkIcon />}
+						bg={'#1F1F32'}
+						colorScheme={'#E0E0EC'}
+						borderRadius={4}
+						variant='solid'>
+            Create invite link
+					</Button>
+				</Flex>
+				<Box h={5} />
+				<Box
+					boxShadow={'lg'}
+					borderRadius={7.5}
+					bg={'white'}
+				>
+					<Table>
+						<Thead>
+							<Tr>
+								{
+									tableHeaders.map((tableHeader) => (
+										<Th
+											key={tableHeader}
+											fontSize={18}
+											fontWeight={'bold'}
+											letterSpacing={-1}
+											textTransform={'none'}
+										>
+											{tableHeader}
+										</Th>
+									))
+								}
+							</Tr>
+						</Thead>
+						{
+							members === undefined ? (
+								<Center>
+									<Loader />
+								</Center>
+							) : members!.length === 0 ? (
+								<Center>
+									<span>
+										{`No ${page === 0 ? '' : 'more '}members!`}
+									</span>
+								</Center>
+							)
+								:
+								(
+									<Tbody>
+										{
+                      members!.map((member: any) => (
+                      	<Tr key={member.id}>
+                      		<Td>
+                      			<Checkbox />
+                      		</Td>
+                      		<Td>
+                      			<Grid>
+                      				<GridItem fontWeight={'bold'}>
+                      					{member.fullName}
+                      				</GridItem>
+                      				<GridItem color={'#9292AF'}>
+                      					{member.actorId}
+                      				</GridItem>
+                      			</Grid>
+                      		</Td>
+                      		<Td>
+                      			<Tag
+                      				bg={'#FFC403'}
+                      				borderRadius={2}
+                      				fontWeight={'bold'}
+                      			>
+                      				{capitalizeFirstLetter(member.accessLevel)}
+                      			</Tag>
+                      		</Td>
+                      		<Td>
+                      			{getFormattedDateFromUnixTimestampWithYear(member.addedAt)}
+                      		</Td>
+                      	</Tr>
+                      ))
+										}
+									</Tbody>
+								)
+						}
+					</Table>
+				</Box>
+				<Box h={2} />
+				<Flex
+					justifyContent={'end'}>
+					<Box
```

Comment:
Same over here, separate component for Pagination handler, maybe something called "PaginatorView"